.github/workflows/wheel_manylinux.yml

```
@@ -0,0 +1,67 @@
+name: wheel - manylinux2014
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  wheel_manylinux:
+    name: manylinux2014
+    runs-on: ${{ matrix.os }}
+    container:
+      image: quay.io/pypa/manylinux2014_x86_64:latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

```suggestion
        os: [ubuntu-18.04]
```

We currently use ubuntu 18.04 in httpstan-wheels

---

18.04 is just the linux that runs the manylinux2014 image.